components/private_channel/rust/ffi/src/private_channel.cpp

```diff
@@ -0,0 +1,78 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "private_channel.hpp"
```

Comment:
includes should not be at the top level `#include "private_channel.hpp"` - this should be `brave/components/private_channel/rust/ffi/src/private_channel.h`

Reply:
I believe speedreader corrected these patterns and should be a good example to follow

Reply:
- [x] add full path to `private_channel.h` include